tests/system/Helpers/DateHelperTest.php

```diff
@@ -32,8 +32,8 @@ public function testNowDefault()
 
     public function testNowSpecific()
     {
-        // Chicago should be two hours ahead of Vancouver
-        $this->assertCloseEnough(7200, now('America/Chicago') - now('America/Vancouver'));
+        // Chicago should be two hours ahead of Vancouver (+- 10 sec)
+        $this->assertCloseEnough(7200, now('America/Chicago') - now('America/Vancouver'), '', 10);
```

Comment:
After all, this test is strange.
It is difficult to accept the *specification* that a 10-second error is not a problem.